cli/lib/kontena/cli/stacks/upgrade_command.rb

```diff
@@ -139,6 +139,11 @@ def execute
         else
           cmd = ['stack', 'install', '--name', stackname]
           cmd.concat ['--parent-name', stack['parent_name']] if stack['parent_name']
+
+          values_from_list.each do |vals_from|
+            cmd.concat ['--values-from', stackname.sub(name, vals_from)]
```

Comment:
What is the `String#sub` doing?

Reply:
I think that's actually supposed to work when using `--values-from-stack`. 

Replacing something like `foo-redis` with `bar-redis` when running with `--values-from bar`.

Reply:
Specs would be nice.